tests/testNonlinearFactorGraph.cpp

```diff
@@ -242,6 +243,49 @@ TEST(testNonlinearFactorGraph, eliminate) {
   EXPECT_LONGS_EQUAL(4, bn->size());
 }
 
+/* ************************************************************************* */
+TEST(testNonlinearFactorGraph, addPrior) {
+  Key k(0);
+
+  // Factor graph.
+  auto graph = NonlinearFactorGraph();
+
+  // Add a prior factor for key k.
+  auto model_double = noiseModel::Isotropic::Sigma(1, 1);
+  graph.addPrior<double>(k, 10.0, model_double);
```

Comment:
do we *need* the <double>. Should be inferred? Same for all tests below.

Reply:
Good point. I removed unnecessary decimals.